api/src/services/users.js

```diff
@@ -569,7 +569,18 @@ const assignCsvCellValue = (data, attribute, value) => {
   if (!attribute.length || attribute.toLowerCase().indexOf(':excluded') > 0) {
     return;
   }
-  data[attribute] = typeof value === 'string' ? value.replace(/^"|"$/g, '').trim() : value;
+
+  if (typeof value !== 'string') {
+    data[attribute] = value;
+    return;
+  }
+
+  data[attribute] = value.replace(/^"|"$/g, '').trim();
+
+  if (['TRUE', 'FALSE'].includes(data[attribute])) {
```

Comment:
Do you think we have to also consider the case of `TRUE` or `FALSE` coming as lower case?

Reply:
I don't think so because spreadsheet editor automatically convert `true` to `TRUE` no matter the capitalization. The Google Sheets used to make the CSV also enforces this by narrowing down this cell value to `TRUE` or `FALSE` only

Reply:
Nice thank you!